src/js/base/module/Clipboard.js

```diff
@@ -18,7 +18,14 @@ export default class Clipboard {
   pasteByEvent(event) {
     const clipboardData = event.originalEvent.clipboardData;
     if (clipboardData && clipboardData.items && clipboardData.items.length) {
-      const item = lists.head(clipboardData.items);
+      // paste img file
+      if (clipboardData.items.length > 1) {
+        const item = clipboardData.items[1];
+      } else {
+        const item = lists.head(clipboardData.items);
+      }
```

Comment:
`error: item is not defined.`

You should defined `item` outside of the `if` block:

```js
const item = clipboardData.items.length > 1
  ? xxx
  : xxx
```